MIP4/MIP4c2-Subproposals/MIP4c2-SP7.md

```diff
@@ -18,15 +18,22 @@ Date of ratification: <yyyy-mm-dd>
 
 This amendment MIP proposes changes to MIP4.
 
-The first being an allowance for MIP amendments to be able to use the Weekly Governance Cycle for the first 3 months after a MIP has passed. This grants new MIPs greater flexibility to be fixed early and fast, improving overall governance efficiency.
+The first change is an allowance for MIP amendments to be able to use the Weekly Governance Cycle for the first 3 months after a MIP has passed. Using the weekly cycle will get an amendment ratified as quickly as 3 weeks instead of 2 months with the original process. This grants new MIPs greater flexibility to be fixed early and fast, improving overall governance efficiency.
```

Comment:
This file shouldn't be in this PR, right? It should have been merged already.

Reply:
good catch, made sure this is up on the master branch. It was out of date there.